src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -339,6 +340,7 @@ public Generator(Stream metadataLibraryStream, GeneratorOptions? options = null,
             this.parseOptions = parseOptions;
 
             this.canCallCreateSpan = this.compilation?.GetTypeByMetadataName(typeof(MemoryMarshal).FullName)?.GetMembers("CreateSpan").Any() is true;
+            this.generateDefaultDllImportSearchPathsAttribute = this.compilation?.GetTypeByMetadataName(typeof(DefaultDllImportSearchPathsAttribute).FullName) is object;
```

Comment:
Drive-by comment: `is not null` is the new `is object`

Reply:
Yes, I've heard of it. My habits haven't changed yet though. :)